src/main/java/games/strategy/engine/lobby/server/db/BadWordController.java

```diff
@@ -69,6 +70,7 @@ void removeBannedWord(final String word) {
       ps.close();
       return rVal;
     } catch (final SQLException sqle) {
+      // SEVERE
       logger.info("Error reading bad words error:" + sqle.getMessage());
```

Comment:
Maybe remove instead? 
`IllegalStateException(sqle.getMessage());` should probably be: `IllegalStateException(sqle)`;

Logging and throwing indicates to me we are going into an unhandled crash. In that case, the stack trace should be logged. In that case the logging we are doing is redundant, and instead we could attach the extra message and propagate the exception eg:
``` throw new IllegalStateException("Error reading bad words error", sqle)```

Reply:
:+1: I was already in the process of implementing your first suggestion, but I think your second suggestion is the right thing to do and will update accordingly.